source/protobuf/session.proto

```diff
@@ -27,6 +27,12 @@ service SessionUtilities {
   rpc ResetServer(ResetServerRequest) returns (ResetServerResponse);
 }
 
+enum SessionInitializationBehavior {
+  SESSION_INITIALIZATION_BEHAVIOR_ATTACH_OR_CREATE = 0;
```

Comment:
Does grpc-device follow https://developers.google.com/protocol-buffers/docs/style?

> The zero value enum should have the suffix UNSPECIFIED.

Reply:
That would make sense. This was intended to be a conversation starter. I prefer being explicit about what unspecified means or allows, but sticking to convention is useful too. Looking at our enum values in various proto files, it seems if 0 is nonsensical then we include it with the suffix `UNSPECIFIED` but if it has a meaningful value then we use a name with meaning. @reckenro thoughts on this one? I'm leaning to sticking with the current name.

Reply:
I've gone with `UNSPECIFIED` since it matches precedent better. My comment about adding it to the Init structures is something we'll likely do in a follow-on PR since it will affect many proto files.